lib/c0/c0sk_internal.h

```diff
@@ -124,6 +125,13 @@ struct c0sk_impl {
     struct cn *c0sk_cnv[HSE_KVS_COUNT_MAX] HSE_L1D_ALIGNED;
 };
 
+
+HSE_ALWAYS_INLINE static bool
```

Comment:
We have been putting this attribute where we usually put `inline` it seems

Reply:
My intent was to make it always inline..